default-views/auth/no-permission.hbs

```diff
@@ -17,8 +17,18 @@
       You are currently logged in as <code>{{webId}}</code>,
       but do not have permission to access <code>{{currentUrl}}</code>.
     </p>
+    <p>
+      <button class="btn btn-primary" onclick="logout()">Log out</button>
```

Comment:
Perhaps we can use `btn-danger` here instead, for consistency with the other logout buttons.

Reply:
Consistency is a good thing. :-)